main.tf

```diff
@@ -77,25 +77,7 @@ data "aws_iam_policy_document" "policy" {
         }
       }
     }
-  }
-
-  statement {
```

Comment:
this is intentional and should stay

Reply:
I have seen that in some edge cases this statement could break connectivity between EC2s and EFSs if the EC2 OS is RHEL or CentOS. I have added a dynamic block with a control variable to add the block. This is true by default and even if it's defined as `null` it gets true because I have set a try inside the dynamic block too.